airgun/entities/contentview_new.py

```diff
@@ -42,12 +43,23 @@ def publish(self, entity_name, values=None):
             view.fill(values)
         view.next.click()
         view.finish.click()
-        view.progressbar.wait_for_result(delay=0.01)
+        # view.progressbar.wait_for_result(delay=0.01)
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
         self.browser.plugin.ensure_page_safe(timeout='5s')
         view.wait_displayed()
         return view.versions.table.read()
 
+    def read_version_table(self, entity_name, version, version_table_name, search_param=None):
+        """Reads a specific table for a CV Version"""
+        view = self.navigate_to(self, 'Version', entity_name=entity_name, version=version)
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        view.wait_displayed()
+        # This allows dynamic access to the proper table
+        getattr(view, version_table_name).table.wait_displayed()
```

Comment:
`version_table_name ` isn't it is a tab rather table, From the https://github.com/SatelliteQE/robottelo/pull/13478/files#diff-53d63aa8641c98a3d50840beb53a9d237755a65ba059f8b7a9571226e793e4b7R94 I see this a tab. I would suggest use `tab_name` instead.